docs/website/root/manual/developer-docs/nodes/mithril-client-library.md

```diff
@@ -104,13 +112,17 @@ async fn main() -> mithril_client::MithrilResult<()> {
 
 :::info
 
-Snapshot download and Certificate chain validation can take quite some time even with a fast computer and network. For those a feedback mechanism is available, more details on it in the [feedback submodule](https://mithril.network/rust-doc/mithril_client/feedback/index.html).
+Snapshot download and certificate chain validation can take quite some time even with a fast computer and network. We have implemented a feedback mechanism for them, more details on it are available in the [feedback sub-module](https://mithril.network/rust-doc/mithril_client/feedback/index.html).
+
+An example of implementation with the crate [indicatif](https://crates.io/crates/indicatif) is available in the [Mithril repository](https://github.com/input-output-hk/mithril/tree/main/mithril-client/examples/snapshot_list_get_show_download_verify.rs). To run it, execute the following command:
 
-An example of implementation with the crate [indicatif](https://crates.io/crates/indicatif) is available in the [Mithril repository](https://github.com/input-output-hk/mithril/tree/main/mithril-client/examples/snapshot_list_get_show_download_verify.rs).
+```bash
+cargo run --example snapshot_list_get_show_download_verify 
```

Comment:
Can you also add the `cd examples` command?

Reply:
It's not needed, it can be run anywhere in the local repo 😉.